Formula/soapysdr.rb

```diff
@@ -0,0 +1,34 @@
+class Soapysdr < Formula
+  desc "Vendor and platform neutral SDR support library"
+  homepage "https://github.com/pothosware/SoapySDR/wiki"
+  url "https://github.com/pothosware/SoapySDR/archive/soapy-sdr-0.8.0.tar.gz"
+  sha256 "cff3b1fb1cd3b88932f37df42f69fc3d8e0ea42f4788b35131ceaf5d1b908ac2"
+  license "BSL-1.0"
+  head "https://github.com/pothosware/SoapySDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.9"
+
+  def install
+    args = []
+    args += ["-DENABLE_PYTHON=OFF"]
+    args += ["-DENABLE_PYTHON3=ON"]
+    args += ["-DCMAKE_MODULE_LINKER_FLAGS_INIT='-undefined dynamic_lookup'"]
+    args += %W[-DSOAPY_SDR_ROOT='#{HOMEBREW_PREFIX}']
+
+    unless build.head?
+      args += ["-DSOAPY_SDR_EXTVER=release"]
+    end
+
+    mkdir "build" do
+      args += std_cmake_args
```

Comment:
Probably just add with during initial assignment to `args` like `args = std_cmake_args + `
```suggestion
```